src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -75,7 +76,7 @@ def __init__(
 
         self.alphas = 1.0 - self.betas
         self.alphas_cumprod = np.cumprod(self.alphas, axis=0)
-        self.one = np.array(1.0)
+        self.negative_alpha_cumprod = np.array(1.0) if do_neg_alpha_one else self.alphas_cumprod[0]
```

Comment:
I'm not sure I understand why this is needed, could you maybe add some comment here explaining why we added it here as this `True` by default. 
Does it affect other pipelines ?

Reply:
yeah maybe we need a better name here indeed. Let me know if you have better ideas? 
In short the story is the following:

For every step we need to know the previous step. Just at the step `t=0` there is no previous step -> so what should we do? 
What is the default here now and what was used before (no breaking change) is the we just set it to 1, but DDIM in stable diffusion instead uses the highest number that exists in the cumulative products of the alphas which is often smaller then one. This can actually make quite a difference in the end

Reply:
Ok I changed it to `clip_alpha_at_one` - think the naming is a bit better no? Wdyt?

Reply:
`set_alpha_to_one`, since it's not necessarily clipping it? (Clipping, to me, suggests that we perform the cumulative product and then make sure it's not larger than 1).

Reply:
changing that actually real quick